lib/AST/ASTDumper.cpp

```diff
@@ -2621,8 +2621,6 @@ class PrintExpr : public ExprVisitor<PrintExpr> {
 
   void printApplyExpr(ApplyExpr *E, const char *NodeName) {
     printCommon(E, NodeName);
-    if (E->isSuper())
```

Comment:
This seems unfortunate, can you ask `E->getArg()->isSuper()` and still print this?